### ISSUE TITLE
Missing file htaccess.txt

### ISSUE THREAD

**Issue by naramski** [opencart/opencart#4837]
When using mod_rewrite, the admin console notices to rename the file htaccess.txt to .htaccess. 

The file cannot be found in the compiled package.

**Comment by danielkerr** [comment]
nope its there

**Comment by naramski** [comment]
Ok. I understand you had the question many time. I have googled first. And searched for this file also. I have the feeling that you are bored by this question, and I understand. I _think_ it could help if the file ".htcacess.txt" was renamed "htaccess.txt". It could help people to find it more quickly and it may help you to receive less question about this problem. Kind regards. Thanks for your work !

**Comment by danielkerr** [comment]
it says htaccess.txt

https://github.com/opencart/opencart/tree/master/upload

**Comment by naramski** [comment]
Indeed. And the name is ".htaccess.txt", starting with a dot. So when you told me it was there, I found it. And again, thanks. But when I tried to find it myself, the starting dot make it invisible for "find . -name htaccess.txt" or "ls". Of course, a simple "ls -la", and the file is there ! I suggest to make it a little bit easier for everyone, and I bet less people will disturb you with this question about SEO.
